### PR TITLE
Create balancedparanthesis.js

### PR DESCRIPTION
**Pull request by KORLA2**
Simple code to find wheather a paranthesis is balanced or not  without using stack

### CODE REVIEW

File: Balanced Paranthesis/balancedparanthesis.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+ function balanced_paranthesis( brackets ){
+ 
+let count=0;
+   
+   for(let i = 0 ; i < brackets.length ; ++i)
+
```
Comment (SaiSudhaV) [inline Balanced Paranthesis/balancedparanthesis.js:6]: Here if and else should be enclosed inside a block or else you will get an exception saying else without a block.

File: Balanced Paranthesis/balancedparanthesis.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+ function balanced_paranthesis( brackets ){
+ 
+let count=0;
```
Comment (SaiSudhaV) [inline Balanced Paranthesis/balancedparanthesis.js:3]: Add spaces before and after operator

File: Balanced Paranthesis/balancedparanthesis.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+   
+   for(let i = 0 ; i < brackets.length ; ++i)
+
+if( brackets[i] === ')' ) count-- ;
+   
+else count++;
```
Comment (SaiSudhaV) [inline Balanced Paranthesis/balancedparanthesis.js:9]: Maintain indentation

**Review by SaiSudhaV — CHANGES_REQUESTED** [review]

**Comment by SaiSudhaV** [comment]
Hello @KORLA2 
Please add the changes suggested. Code looks great just need to beautify it.

Thank you
Sudha

**Comment by KORLA2** [comment]
Hello I have changed have a look

**Comment by KORLA2** [comment]
Hello I have added spaces

**Comment by KORLA2** [comment]
Any updates on my PR

File: Balanced Paranthesis/balancedparanthesis.js (hunk 1 above)
Comment (SaiSudhaV) [reply Balanced Paranthesis/balancedparanthesis.js:6]: I mentioned to add to for block not for if and else. I have explained the cause for not enclosing for loop in block. Please add that.

**Review by SaiSudhaV — CHANGES_REQUESTED** [review]

**Comment by KORLA2** [comment]
Hello I have made changes

**Review by SaiSudhaV — APPROVED** [review]

**Pull request merged**